src/bridge/cockpitpackages.c

```diff
@@ -918,7 +918,7 @@ package_content (CockpitPackages *packages,
               cockpit_web_response_error (response, 404, NULL, NULL);
               goto out;
             }
-          else if (package->unavailable)
+          else if (package && package->unavailable)
```

Comment:
e. g. it's not really clear if it really makes sense to not send an error here if `package` is NULL. (Might be, I didn't read enough of the surrounding code to fully understand this).